koku/api/settings/test/tags/test_api.py

```diff
@@ -68,12 +68,20 @@ def test_get_tags(self):
 
     def test_get_tags_filtering(self):
         test_matrix = (
+            # Custom syntax
             {"filter": {"filter[source_type]": "AWS"}, "key": "source_type", "expected": {"AWS"}},
             {"filter": {"filter[source_type]": ["AWS", "GCP"]}, "key": "source_type", "expected": {"AWS", "GCP"}},
             {"filter": {"filter[key]": "env"}, "key": "key", "expected": {"env", "Environment"}},
             {"filter": {"filter[enabled]": True}, "key": "enabled", "expected": {True}},
             {"filter": {"filter[enabled]": "true"}, "key": "enabled", "expected": {True}},
             {"filter": {"filter[enabled]": "false"}, "key": "enabled", "expected": {False}},
+            # DRF Syntax
+            {"filter": {"source_type": "AWS"}, "key": "source_type", "expected": {"AWS"}},
+            {"filter": {"provider_type": ["AWS", "GCP"]}, "key": "source_type", "expected": {"AWS", "GCP"}},
```

Comment:
Don't believe we need both source_type and provider_type. Why support both?

Reply:
For completeness. Long term, I think we should change the model to avoid needing this.